
From: <janitor@sternwelten.at>

  remove unused #include <linux/version.h>
  Old ifdefs were removed that used it's definition.

Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/block/cpqarray.c |   14 --------------
 1 files changed, 14 deletions(-)

diff -puN drivers/block/cpqarray.c~janitor-cpqarray-remove-unused-include drivers/block/cpqarray.c
--- 25/drivers/block/cpqarray.c~janitor-cpqarray-remove-unused-include	Thu Sep 23 15:38:16 2004
+++ 25-akpm/drivers/block/cpqarray.c	Thu Sep 23 15:38:16 2004
@@ -21,7 +21,6 @@
  */
 #include <linux/config.h>	/* CONFIG_PROC_FS */
 #include <linux/module.h>
-#include <linux/version.h>
 #include <linux/types.h>
 #include <linux/pci.h>
 #include <linux/bio.h>
@@ -732,7 +731,6 @@ static void *remap_pci_mem(ulong base, u
 }
 
 #ifndef MODULE
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,3,13)
 /*
  * Config string is a comma separated set of i/o addresses of EISA cards.
  */
@@ -749,18 +747,6 @@ static int cpqarray_setup(char *str)
 
 __setup("smart2=", cpqarray_setup);
 
-#else
-
-/*
- * Copy the contents of the ints[] array passed to us by init.
- */
-void cpqarray_setup(char *str, int *ints)
-{
-	int i;
-	for(i=0; i<ints[0] && i<8; i++)
-		eisa[i] = ints[i+1];
-}
-#endif
 #endif
 
 /*
_
